css-brochure-form/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header><h1>LEARN MORE</h1></header>
+
+    <form>
+      <div id="content">
+        <div>
+          <label for="email" class="label">COMPANY EMAIL</label>
+          <input
+            type="email"
+            id="email"
+            placeholder="sally@company.co"
+            class="input" />
+        </div>
+        <div>
+          <label for="name" class="label">NAME</label>
+          <input
+            type="text"
+            id="name"
+            placeholder="Sally Smith"
+            class="input" />
+        </div>
+        <div>
+          <label for="location" class="label">WHERE ARE YOU HIRING?</label>
+          <input
+            type="text"
+            id="location"
+            placeholder="City, County"
+            class="input" />
+        </div>
+        <div>
+          <label for="option" class="label"
+            >HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
+          >
+          <select id="option" class="input">
+            <option value="1-2">1-2 Engineers</option>
+            <option value="2-4">2-4 Engineers</option>
+            <option value="3-6">3-6 Engineers</option>
+            <option value="6+">6+ Engineers</option>
+          </select>
+          <br />
```

Comment:
You should not be using `<br />`. It's super bad practice. If you're using this for spacing reasons, you should be using `margin` instead